hw/ip/otbn/dv/otbnsim/sim/alert.py

```diff
@@ -4,11 +4,17 @@
 
 from typing import Optional
 
-# A copy of the list of error codes. This also appears in otbn.hjson and
+# A copy of the list of error codes. This also appears in the documentation and
 # otbn_pkg.sv: we should probably be generating them from the hjson every time.
-ERR_CODE_NO_ERROR = 0
-ERR_CODE_BAD_DATA_ADDR = 1
-ERR_CODE_CALL_STACK = 2
+ERR_CODE_NO_ERROR = 0x0
+ERR_CODE_BAD_DATA_ADDR = 0x1
+ERR_CODE_BAD_INSN_ADDR = 0x2
+ERR_CODE_CALL_STACK = 0x3
+ERR_CODE_ILLEGAL_INSN = 0x4
+ERR_CODE_LOOP = 0x5
+ERR_CODE_FATAL_IMEM_ECC = 0x80
+ERR_CODE_FATAL_DMEM_ECC = 0x81
+ERR_CODE_FATAL_REG_ECC = 0x82
```

Comment:
Note that ECC has been dropped from these names elsewhere

Reply:
Ah, good point, thanks.